assets/js/googlesitekit/widgets/components/WidgetHeaderCTA.js

```diff
@@ -38,12 +38,13 @@ const WidgetHeaderCTA = ( { href, label, external } ) => (
 );
 
 WidgetHeaderCTA.propTypes = {
-	href: PropTypes.string.isRequired,
+	href: PropTypes.string,
 	label: PropTypes.string.isRequired,
 	external: PropTypes.bool,
 };
 
 WidgetHeaderCTA.defaultProps = {
+	href: '#',
 	external: true,
 };
```

Comment:
This looks problematic, why did you add it? We should never use `#` for a `href` attribute; also why is it not required?

Reply:
The `getServiceReportURL` selector returns undefined at the beginning. It causes console errors because `WidgetHeaderCTA` requires a valid URL. As a workaround, I have decided to drop `.isRequired` for the `href` attribute and set `#` by default.

Reply:
@eugene-manuilov Can you use an empty string as default instead? That's what `SourceLink` does too.

Reply:
Ok, I'll update it.

Reply:
Updated.